panolint-rails-rubocop.yml

```diff
@@ -12,6 +12,8 @@ inherit_gem:
 
 require:
   - rubocop-performance
```

Comment:
I get warnings when I try to do plugins with `rubocop-performance`. I think this has to do with the version we're using

Reply:
I'm confused. Aren't `rubocop-performance` and `rubocop-rspec` already coming from `panolint-ruby`? `panolint-rails` should be additive, not a totally separate dependency path.